packages/stryker-mocha-runner/package.json

```diff
@@ -35,10 +35,12 @@
   },
   "homepage": "https://github.com/stryker-mutator/stryker/tree/master/packages/stryker-mocha-runner#readme",
   "dependencies": {
+    "fast-glob": "^2.1.0",
```

Comment:
We might want to find an alternative for this in the future because it has a lot of dependencies.

Reply:
Yeah, but let's use this for now. We can experiment later.